proteus/mprans/RANS2P.py

```diff
@@ -476,10 +534,10 @@ def attachModels(self, modelList):
                     self.ebq_n = None
                 self.ebqe_n = modelList[self.LS_model].ebqe[('grad(u)', 0)]
             else:
-                self.q_phi = 10.0 * numpy.ones(self.model.q[('u', 1)].shape, 'd')
-                self.phi_dof = -numpy.ones_like(self.model.u[0].dof)
-                self.ebqe_phi = 10.0 * numpy.ones(self.model.ebqe[('u', 1)].shape, 'd')
-                self.bc_ebqe_phi = 10.0 * numpy.ones(self.model.ebqe[('u', 1)].shape, 'd')
+                self.q_phi = -10.0 * numpy.ones(self.model.q[('u', 1)].shape, 'd')
```

Comment:
What's the factor of 10 for in general? And what's the significance of the minus sign

Reply:
Those are just to give reasonable defaults that push the level set firmly into a single phase.